### ISSUE TITLE
Apply for f-droid repo

### ISSUE THREAD

**Issue by veganomy** [stsaz/phiola#59]
I think it's time to reconsider switching from Izzyondroid to f-droid. Many seem to trust only the official repo as a means to install open source applications.

**Comment by stsaz** [comment]
I created a request here: https://gitlab.com/fdroid/rfp/-/issues/3143
But I wouldn't expect that it will be resolved in the near future.

**Comment by stsaz** [comment]
Somebody has to spend the time preparing a special meta file so that the f-droid system can build and publish the app.  It's not trivial, as far as I can see, for the project like phiola that depends on lots of native code (i.e. code in C language) and uses `make` build system.

**Comment by veganomy** [comment]
Hi. Can you look into this request ? I think you need to resolve some of the things that F-Droid bot shows :

https://gitlab.com/fdroid/rfp/-/work_items/3143#note_2549966214

**Comment by stsaz** [comment]
Yeah, these issues were resolved long ago, but the real work is to prepare an f-droid automated script.  Just recently, I thought about doing it, and the plan is:
* to set up an auto-build script for GH first (so it can build, sign and upload a fully working phiola/Android package)
* and then adapt the script for f-droid.
